vue_application/Ransomware Hunter/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
```

Comment:
You excluded node_modules in the .gitignore but because it's been checked into the repo, it's going to still be updated. Given the state of the project, I don't think it'd be an issue for the future so it's up to you if you wanna fix it.